src/components/grid/grid-container/grid-container.component.js

```diff
@@ -38,27 +20,7 @@ GridContainer.propTypes = {
   /** Defines the Components to be rendered within the GridContainer. Requires a GridItem */
   children: PropTypes.oneOfType([gridItemType, PropTypes.arrayOf(gridItemType)])
     .isRequired,
-  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default padding */
-  p: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default padding-left */
-  pl: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default padding-right */
-  pr: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default padding-top */
-  pt: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default padding-bottom */
-  pb: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /**
-   * Any valid CSS value or a number to be multiplied by base spacing unit (8).
-   * Overrides default horizontal paddings
-   * */
-  px: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /**
-   * Any valid CSS value or a number to be multiplied by base spacing unit (8).
-   * Overrides default vertical paddings
-   * */
-  py: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  /** Any valid CSS value to override default grid-gap */
-  gridGap: PropTypes.string,
+  ...propTypes.padding,
+  ...propTypes.gridGap,
```

Comment:
This isn't actually valid, the available type categories for the `@styled-system/prop-types` package are shown here: https://github.com/styled-system/styled-system/tree/master/packages/prop-types

Also if you look at the prop table for `GridContainer` in storybook you can see that `gridGap` is no longer there in this branch.

Saying as we're only using the `gridGap` prop from `styled-system` here, I would just define it as:

```jsx
/** Any valid CSS value to override default grid-gap */
gridGap: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

and then it will be correct, and be present in the props table in the docs